spec/features/search_page_spec.rb

```diff
@@ -121,165 +79,88 @@
+          build(:dataset, title: 'A fairly interesting dataset'))
 
     search_for('"Very interesting"')
-
     assert_search_results_headings('A very interesting dataset')
   end
 
   scenario 'Searching for a malformed phrase' do
-    index(DatasetBuilder.new.with_title('A very interesting dataset').build,
-          DatasetBuilder.new.with_title('A fairly interesting dataset').build)
+    index(build(:dataset, title: 'A very interesting dataset'),
+          build(:dataset, title: 'A fairly interesting dataset'))
 
     search_for('"Very interesting')
 
-    assert_search_results_headings(
-      'A very interesting dataset',
-      'A fairly interesting dataset'
-    )
-  end
-
-  scenario 'Searching with a plural query' do
-    index(DatasetBuilder.new.with_title('Maps').build)
-
-    search_for('Map')
-
-    assert_search_results_headings('Maps')
+    assert_search_results_headings('A very interesting dataset',
+                                   'A fairly interesting dataset')
   end
 
   scenario 'Search with a possessive query' do
-    index(DatasetBuilder.new.with_title('Government').build)
-
+    index(build(:dataset, title: 'Government'))
     search_for("Government's")
-
     assert_search_results_headings('Government')
   end
 
-  scenario 'Search with lowercase query' do
-    index(DatasetBuilder.new.with_title('DEFRA').build)
-
-    search_for("defra")
-
-    assert_search_results_headings('DEFRA')
-  end
-
   scenario 'Prominence of datasets based on organisation category' do
-    ministerial_dataset = DatasetBuilder
-                            .new
-                            .with_ministerial_department(
-                              'Department for Environment Food & Rural Affairs'
-                            )
-                            .build
-
-    non_ministerial_dataset = DatasetBuilder
-                                .new
-                                .with_non_ministerial_department(
-                                  'Forestry Commission'
-                                )
-                                .build
-
-    local_council_dataset = DatasetBuilder
-                              .new
-                              .with_local_council(
-                                'Plymouth City Council'
-                              )
-                              .build
-
-    non_departmental_public_body_dataset = DatasetBuilder
-                                             .new
-                                             .with_non_departmental_public_body(
-                                               'English Heritage'
-                                             )
-                                             .build
-
-    index(
-      ministerial_dataset,
-      non_ministerial_dataset,
-      local_council_dataset,
-      non_departmental_public_body_dataset
-    )
+    dataset1 = build :dataset, organisation: build(:organisation, :raw)
```

Comment:
Note: this test is actually passing by chance. The organisation boost functionality is broken.

Reply:
If you change the order the datasets are indexed in, the test fails.